test/malli/instrument/cljs_test/indirect_fn.cljs

```diff
@@ -0,0 +1,28 @@
+(ns malli.instrument.cljs-test.indirect-fn
+  "Bug to do with when things are instrumented
+   This declared functions here to be used in cljs_test"
+  (:require [malli.core :as m]
+            [malli.experimental :as mx]
+            [malli.instrument.cljs-test.defs
+             :as defs
+             :refer [defs-small-int defs-int-arg]]))
+
+;; There's a matrix of things to test here
+;; Calling either arg or return type from another namespace
+;;  vs whether doing it as a refer or as a namespaced var
+;; This hits all 4
```

Comment:
For completeness, can you add the scenario where the schema is a core var like `int?`